docs/source/markdown/options/network.md

```diff
@@ -10,9 +10,9 @@ Valid _mode_ values are:
 
 - **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootful containers. It is possible to specify these additional options:
     - **alias=**_name_: Add network-scoped alias for the container.
-    - **ip=**_IPv4_: Specify a static ipv4 address for this container.
-    - **ip=**_IPv6_: Specify a static ipv6 address for this container.
-    - **mac=**_MAC_: Specify a static mac address for this container.
+    - **ip=**_IPv4_: Specify a static IPv4 address for this container.
+    - **ip6=**_IPv6_: Specify a static IPv6 address for this container.
```

Comment:
technically it does not matter and you can very well set `ip=<some ipv6 address>`, the backend code mixes them together anyway. But for documentation purposes this makes sense.

Reply:
It does matter in my test with Podman Desktop 5.0.2. on MacOS. With `ip` and an IPv6 address, the container doesn't get the address assigned and with `ip6`, it does.

Reply:
If you use a GUI then why does the cli syntax apply at all?

Reply:
Who says I use a GUI? I'm using the CLI, just listed the product.

Reply:
Because if you use the cli you cannot say that `ip` doesn't work for a ipv6 address, the code is literally the same no matter of `ip` or `ip6`
https://github.com/containers/podman/blob/9ac6d9de45ab18591a2bc38994ebda1d0d2bd2d0/pkg/specgen/namespaces.go#L448

Reply:
Maybe you can also test it, looks can be deceptive.